frontend/src/app/modules/grids/grid/grid.component.html

```diff
@@ -19,12 +19,14 @@
     <div class="grid--area-content widget-box"
          cdkDrag
          (cdkDragStarted)="drag.start(area)"
-         (cdkDragDropped)="drag.drop()">
+         (cdkDragDropped)="drag.drop()"
+         [cdkDragDisabled]="browserDetector.isMobile">
 
       <span *ngIf="drag.isDraggable"
             class="grid--area-drag-handle
                    icon
-                   icon-drag-handle hidden-for-mobile"
+                   icon-drag-handle"
+            [hidden]="browserDetector.isMobile"
```

Comment:
I made this change because the handle should be shown, if dragging is possible. Due to the change you can also drag and drop on small screens, but only on desktop, not mobile devices. The class "hidden-for-mobile" would always hide the handle for small screens even if dragging would be enabled.

Reply:
But it doesn't work on mobile sized browsers:
![drag_mobile](https://user-images.githubusercontent.com/7457313/73543091-79c38180-4436-11ea-96e1-c3112ccb8c71.gif)

And I guess it cannot work because of the changes we make to the grid on mobile screen size. Without having debugged it, I assume that the location in the grid cannot be determined correctly and therefore it does not work. I think pragmatism is needed here, so it's better to simply hide it on mobile screen size.

Reply:
Oh, you are right. I didn't remember the grid changes. I will add the class again.